src/selectfield/selectfield.scss

```diff
@@ -50,17 +50,32 @@ $selectfield-error-color: unquote("rgb(222, 50, 38)") !default;
   border-radius: 0;
   -webkit-appearance: none;
   -moz-appearance: none;
+  -ms-appearance: none;
   appearance: none;
 
+  &:-moz-focusring {
+    color: transparent;
+    text-shadow: 0 0 0 #000000;
+  }
+
   .mdl-selectfield.is-focused & {
     outline: none;
   }
 
+  &:focus::-ms-value {
```

Comment:
Could you re-organize the order of the sub-elements so that you have all the peusdo-elements one after another, etc? it's easier to read